lib/rules/aria-hidden-body.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "aria-hidden-body",
+  "selector": "body",
+  "excludeHidden": false,
+  "tags": [
+    "cat.aria",
+    "wcag2a"
```

Comment:
Sorry, missed that, but you're missing a tag for the wcag criterion. i suggest wcag412.